### PR TITLE
gupnp 0.20.14

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

**Comment by DomT4** [comment]
```
libtool: compile:  clang -DHAVE_CONFIG_H -I. -I.. -D_REENTRANT -I/usr/local/Cellar/glib/2.44.1/include/glib-2.0 -I/usr/local/Cellar/glib/2.44.1/lib/glib-2.0/include -I/usr/local/opt/gettext/include -I/usr/local/Cellar/libsoup/2.48.1/include/libsoup-2.4 -I/usr/local/Cellar/gssdp/0.14.8/include/gssdp-1.0 -I/usr/include/libxml2 -I.. -g -O2 -c gupnp-marshal.c  -fno-common -DPIC -o .libs/gupnp-marshal.o
guul.c:25:13: fatal error: 'uuid.h' file not found
#   include <uuid.h>
            ^
1 error generated.
make[2]: *** [guul.lo] Error 1
make[2]: *** Waiting for unfinished jobs....
```

Maybe needs a `ossp-uuid` dep?

File: Library/Formula/gupnp.rb [hunk 1]
```diff
@@ -27,6 +27,11 @@ class Gupnp < Formula
   depends_on "libsoup"
   depends_on "gssdp"
 
+  patch :p0 do
+    url "https://trac.macports.org/export/136642/users/devans/GNOME-3/stable/dports/net/gupnp/files/patch-osx-uuid.diff"
```
Comment (DomT4) [inline Library/Formula/gupnp.rb]: Any idea if this ever got reported upstream?

Comment (nijikon) [inline Library/Formula/gupnp.rb]: Does not look like it, no changes in the codebase of gupnp.

Comment (DomT4) [inline Library/Formula/gupnp.rb]: Sigh. Thanks for kicking it upstream, you may need to kick it to them directly as well, I'm not sure how actively they check the Github mirrors. Happy to merge it with the reference to the PR though.

**Comment by DomT4** [comment]
Why'd we change to the `snapshot` URL here? Is there no stable release of `0.20.14` or did they move it? The checksum fell over apparently:

```
Error: SHA256 mismatch
Expected: 77ffb940ba77c4a6426d09d41004c75d92652dcbde86c84ac1c847dbd9ad59bd
Actual: cd6c1aaecfe06c89b5fcd758e3cbe662611e1e1f47c449836570e9fa43e22ebf
```

**Comment by nijikon** [comment]
@DomT4 sorry, new sha256 did not commit. I changed this cause this is the official link on the page and it is https.

**Comment by DomT4** [comment]
> I changed this cause this is the official link on the page and it is https.

Where is this link? Navigating to https://wiki.gnome.org/Projects/GUPnP#Documentation seems to have a link pointing at http://download.gnome.org/sources/ in the `Download` section?

**Comment by nijikon** [comment]
@DomT4 you are right, fixed.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/09452a5cca0c925bbe07d948dabe38f8ca2fa753. Thanks!

**Pull request closed without merging**